testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/mock/JobWorkerMock.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.process.test.api.mock;
+
+import io.camunda.client.CamundaClient;
+import io.camunda.client.api.worker.JobHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JobWorkerMock {
+
+  private final String jobId;
```

Comment:
:wrench: We should rename the field to `jobType`. A job has a type and a job worker fetches jobs for a given type.